be/src/exprs/block_bloom_filter.hpp

```diff
@@ -177,7 +180,7 @@ class BlockBloomFilter {
 
 #endif
     // Size of the internal directory structure in bytes.
-    int64_t directory_size() const { return 1ULL << log_space_bytes(); }
+    size_t directory_size() const { return 1ULL << log_space_bytes(); }
```

Comment:
Why use platform related size here? Using `uint64_t` is better.

```suggestion
    uint64_t directory_size() const { return 1ULL << log_space_bytes(); }
Write Preview
```

Reply:
It's actually s size_t and we use it to compare with other variables with size_t type everywhere. So I just want to keep consistency with other variables here.